beacon_node/beacon_chain/src/block_verification_types.rs

```diff
@@ -122,23 +122,10 @@ impl<E: EthSpec> RpcBlock<E> {
         // Treat empty blob lists as if they are missing.
         let blobs = blobs.filter(|b| !b.is_empty());
 
-        if let (Some(blobs), Ok(block_commitments)) = (
-            blobs.as_ref(),
-            block.message().body().blob_kzg_commitments(),
-        ) {
-            if blobs.len() != block_commitments.len() {
-                return Err(AvailabilityCheckError::MissingBlobs);
-            }
-            for (blob, &block_commitment) in blobs.iter().zip(block_commitments.iter()) {
-                let blob_commitment = blob.kzg_commitment;
-                if blob_commitment != block_commitment {
-                    return Err(AvailabilityCheckError::KzgCommitmentMismatch {
```

Comment:
We don't need to check the commitment equivalence anymore because:
- We check the inclusion proof of each blob
- We match blobs and blocks by block root

Therefore it's impossible for the commitment to not be equivalent